mlflow/exceptions.py

```diff
@@ -143,4 +143,7 @@ class InvalidUrlException(MlflowException):
 class UnsupportedMultipartUploadException(MlflowException):
```

Comment:
Can we mark this as private to avoid accidentally documenting this class?

Reply:
sure